### ISSUE TITLE
A comment on recipe balance.

### ISSUE THREAD

**Issue by chrisbecke** [Azanor/thaumcraft-suggestions#7]
While the thaumcraft plates are now in line with other mod's plate recipes, the 5.2.3 gear recipes are far cheaper than gears from other mods, where the standard seems to be 4 ingots arranged around a center core.

I assume these gears and plates are forge dictionary compatible? 

At any rate, while Thaumcraft needs its own balance, when there is strong prior work that is being referenced, deviating from the established material cost expectations is going to cause balance issues in modpacks.

**Comment by Fortanono** [comment]
Support. Makes sense.

**Comment by Azanor** [comment]
A good point.

**Comment by Fortanono** [comment]
If you end up going this way, I'd say an iron plate in the middle will stop you from coming into conflict with other mods.

**Comment by MastaC729** [comment]
The following was off of an issue I accidentally posted to the wrong Github page:

I think it would be nice to be able to disable Thaumcraft's plate recipes, either through a config option or detection of IC2 or other mods that create plates for machinery. Obviously plates would be enabled by default, but modpack users could easily disable the recipes so that players can't short circuit the process of making plates in mods like IC2 where it involves the hammer, the metal former, or the Smeltery.

**Comment by Fortanono** [comment]
Isn't Minetweaker available for 1.8.9? It would be easy enough to remove the recipes that way.

**Comment by MastaC729** [comment]
Yes, and that works just fine. But it would be cool to have it built into Thaumcraft, since my modpack would use Minetweaker just to modify that one set of recipes.
